scipy/integrate/_quadpack_py.py

```diff
@@ -330,6 +334,29 @@ def quad(func, a, b, args=(), full_output=0, epsabs=1.49e-8, epsrel=1.49e-8,
         Clenshaw-Curtis integration is used on those intervals containing the
         point :math:`x = c`.
 
+    ** Integration of Complex Function of a Real Variable **
```

Comment:
This is not rendering as intended.

![image](https://user-images.githubusercontent.com/6570539/194391203-d0df7807-19a6-4248-927c-919cf2bd759c.png)

```suggestion
    **Integration of Complex Function of a Real Variable**
```